qml/components/MainPage.qml

```diff
@@ -151,6 +168,13 @@ Page {
                 deleteIcon.visible = true
             }
 
+            onClicked: {
+                var url = model.imgout
```

Comment:
At this point we need the correct path to make this work. Not the image://document/id path. Is there a way to get the correct path?

Reply:
The ImageProcessing plugin gives you all the URLs needed to export PDFs or images: https://github.com/jonnius/camera-scanner/blob/master/plugins/ImageProcessing/ImageProcessing.h#L31